code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -386,7 +386,7 @@
 	DELAY 30
 	SAY You have been selected out of $)@! potential candidates for this post!
 	DELAY 30
-	SAY Nanotrasen is pleased to have you working in one of the many top-of-the-line research posts within the $%@!! sector!
+	SAY Horizons is pleased to have you working in one of the many top-of-the-line research posts within the $%@!! sector!
```

Comment:
SAY Horizons is pleased to have you working in one of the many exploratory posts within the $%@!! sector!